k8s/upgrades/0.8.2-0.9.0/cstor/cstor-volume-update-082-090.yaml

```diff
@@ -231,8 +233,11 @@ spec:
   post: |
     {{- jsonpath .JsonResult "{.items[*].metadata.name}" | trim | saveAs "listTargetDeployment.targetDeploymentName" .TaskResult | noop -}}
     {{- .TaskResult.listTargetDeployment.targetDeploymentName | notFoundErr "volume target deployment not found" | saveIf "listTargetDeployment.notFoundErr" .TaskResult | noop -}}
-    {{- jsonpath .JsonResult "{.items[0].metadata.labels.openebs\\.io/version}" | trim | saveAs "listTargetDeployment.labels" .TaskResult | noop -}}
-    {{- jsonpath .JsonResult "{.items[0].metadata.labels.openebs\\.io/persistent-volume-claim}" | trim | saveAs "listTargetDeployment.pvcName" .TaskResult | noop -}}
+    {{- jsonpath .JsonResult "{.items[*].metadata.labels.openebs\\.io/version}" | trim | saveAs "listTargetDeployment.labels" .TaskResult | noop -}}
+    {{- jsonpath .JsonResult "{.items[*].metadata.labels.openebs\\.io/persistent-volume-claim}" | trim | saveAs "listTargetDeployment.pvcName" .TaskResult | noop -}}
+
+    {{- jsonpath .JsonResult "{.items[*].spec.replicas}" | trim | saveAs "listTargetDeployment.replicaCount" .TaskResult | noop -}}
+    {{- .TaskResult.listTargetDeployment.replicaCount | notFoundErr "unknown no.of replicas in cstor-target deployment" | saveIf "listTargetDeployment.notFoundErr" .TaskResult | noop -}}
```

Comment:
verifyErr instead of notFoundErr

`Replicas not set for cstor target deployment`

Reply:
I Will update the error message